src/typical.jl

```diff
@@ -54,17 +54,24 @@ end
 
 function typify(refgrid, model_formula::FormulaTerm,
                 model_matrix::AbstractMatrix; typical=mean)
-    model_terms = terms(model_formula)
-
     effects_terms = Term.(Tables.columnnames(refgrid))
 
     # creates a MatrixTerm (and should work for things like MixedModels) which
     # should correspond to the model_matrix
     matrix_term = get_matrix_term(model_formula.rhs)
-
     typical_terms = Dict()
-    for term in terms(matrix_term)
-        if !any(et -> StatsModels.symequal(et, term), effects_terms)
+
+    urterms = terms(matrix_term)
+    nonfunc_terms = [filter(tt -> !isa(tt, FunctionTerm), matrix_term.terms)...]
```

Comment:
```suggestion
    nonfunc_terms = [tt for tt in matrix_term.terms if !isa(tt, FunctionTerm)]
```

or 

```suggestion
    nonfunc_terms = collect(filter(tt -> !isa(tt, FunctionTerm), matrix_term.terms))
```

to avoid splatting?